[Workflow management UI changes 4] Various UI tweaks to workflow management

kaedroho opened the pull request:
This pull request contains various UI improvements to the workflow management interface.

Separated from: https://github.com/wagtail/wagtail/pull/6157

zerolab commented on wagtail/admin/urls/workflows.py:
```diff
@@ -5,13 +5,13 @@
+    path('workflows/disable/<int:pk>/', workflows.Disable.as_view(), name='disable'),
+    path('workflows/edit/<int:pk>/', workflows.Edit.as_view(), name='edit'),
+    path('workflows/remove/<int:page_pk>/', workflows.remove_workflow, name='remove'),
+    path('workflows/remove/<int:page_pk>/<int:workflow_pk>/', workflows.remove_workflow, name='remove'),
```
For consistency, `wagtail/admin/urls/__init__.py` should retain the `workflows/` prefix, and these `workflow/` (since all but the index one apply to one workflow)

zerolab commented on wagtail/admin/templates/wagtailadmin/workflows/index.html:
```diff
@@ -13,49 +13,66 @@ <h1 class="icon icon-{{ view.header_icon }}">
+                {% endif %}
+
+                {% trans "Show disabled workflows" %}
+            </label>
```
2 issues with this:

1. the label gets wrapped weirdly because of the 16% width  rule on it
![Screenshot 2020-06-25 at 14 40 29](https://user-images.githubusercontent.com/31622/85734638-12fc7880-b6f5-11ea-9b55-68ab758c86ae.png)
Changing the text to "Show disabled" sorts this out
2. The inline JS. I suspect it may trip CSP (a la #1288)

kaedroho commented on wagtail/admin/templates/wagtailadmin/workflows/edit.html:
```diff
@@ -54,17 +53,21 @@
+                <ul>
+                    <li class="actions">
+                        {% block form_actions %}
+                            <div class="dropdown dropup dropdown-button match-width">
```
I have a commit for this, but it depends on changes in your UI tweaks branch: https://github.com/kaedroho/wagtail/compare/feature/workflow-management-ui-tweaks...feature/workflow-management-move-disable-button

zerolab replied:
more like #6151

kaedroho commented:
@zerolab Just noticed that I missed the tasks index, I've added it there in my latest fixup: https://github.com/wagtail/wagtail/pull/6177/commits/a141562499e044c3ded820aa530b84fcdbe4753f

Also moved the checkbox outside the `{% if workflows %}` so it still displays if there are no active workflows

zerolab commented:
Nice one @kaedroho
